ThunderCats/UITextField+TCAdditions.m

```diff
@@ -40,4 +40,27 @@ - (void)tc_setLeftAndRightPadding:(CGFloat)paddingWidth
     self.rightViewMode = UITextFieldViewModeUnlessEditing;
 }
 
+- (void)tc_addToolbarWithNextButton:(UITextField * __tc_nonnull)nextTextField
+                   toolbarColor:(UIColor * __tc_nonnull)toolbarColor
+{
+    [self tc_addToolbarWithButton:@"Next" toolbarColor:toolbarColor target:nextTextField action:@selector(becomeFirstResponder)];
+}
+
+- (void)tc_addToolbarWithButton:(NSString * __tc_nonnull)buttonTitle toolbarColor:(UIColor * __tc_nonnull)toolbarColor target:(id)target action:(SEL)selector
```

Comment:
I'd prefer to see the arguments on separate lines here.  This is done inconsistently throughout this file.  I'm kind of okay with inconsistent, but we should prefer the methods with *more* arguments to be broken onto multiple lines with colons aligned, and methods with fewer arguments could be okay on a single line, but it looks like we've done the opposite.